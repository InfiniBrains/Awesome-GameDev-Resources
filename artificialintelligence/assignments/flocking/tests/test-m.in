0.000 3.000 1.000 0.000 1.000 2.000 1.100 5
0.500 0.500 0.000 0.000
-0.500 -0.500 0.000 0.000
-0.500 0.500 0.000 0.000
0.500 -0.500 0.000 0.000
0.000 0.000 0.000 0.000
0.125
0.125
0.125
