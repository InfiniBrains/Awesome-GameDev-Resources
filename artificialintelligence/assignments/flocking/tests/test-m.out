0.522 0.522 0.177 0.177
-0.522 -0.522 -0.177 -0.177
-0.522 0.522 -0.177 0.177
0.522 -0.522 0.177 -0.177
0.000 0.000 0.000 0.000
0.569 0.569 0.378 0.378
-0.569 -0.569 -0.378 -0.378
-0.569 0.569 -0.378 0.378
0.569 -0.569 0.378 -0.378
0.000 0.000 0.000 0.000
0.645 0.645 0.607 0.607
-0.645 -0.645 -0.607 -0.607
-0.645 0.645 -0.607 0.607
0.645 -0.645 0.607 -0.607
0.000 0.000 0.000 0.000
